### PR TITLE
change the title and add favicon

### LINKED ISSUE #3: Change `title` in `index.html`
In `index.html` the `title` is a generic one. We need to change it to something meaningful.

### LINKED ISSUE #4: Add a favicon to the project
Our project needs a [favicon](https://en.wikipedia.org/wiki/Favicon) for it to look good. Please add one.

### PR DESCRIPTION
**Pull request by M-Hajjar**
here you may see I have changed the title and added a favicon

### CODE REVIEW

**Review by azizsigar — APPROVED** [review]

File: index.html
Comment (imtiaz101325) [inline index.html]: Please remove these.

**Review by imtiaz101325 — CHANGES_REQUESTED** [review]
Looks good but you have some unnecessary code.

**Comment by imtiaz101325** [comment]
Once finished this closes #3 and #4 .

**Comment by beimnet11** [comment]
this is fine to me

**Comment by imtiaz101325** [comment]
@beimnet11 you were not supposed to close this PR.

**Comment by beimnet11** [comment]
@imtiaz101325 sorry it was by mistake, i was exploring through github.

**Comment by M-Hajjar** [comment]
as a result here we have added only a title to the page 
and also a favicon

**Comment by imtiaz101325** [comment]
@M-Hajjar you were supposed to wait for more reviews before merging 😅
For instance I see you removed the `li` items but the `ul` remains.

**Comment by M-Hajjar** [comment]
oh my bad i thought we agreed on this one  
will be more careful for the rest

**Pull request merged**
